### PR TITLE
[#657] Set up SDK Sentry

### LINKED ISSUE #657: Add Sentry support to the SDK
Send Sentry messages to a configured Sentry instance.

* [x] Make Sentry DSN an option in the SDK constructor
* [x] Adjust SDK snippet to include the Sentry DSN
* [x] Everything should also work without Sentry DSN configured
* [x] Add envvar to deployment/settings to include this public (!) Sentry DSN

### PR DESCRIPTION
**Pull request by SilviaAmAm**
Fixes #657 

SDK PR: https://github.com/open-formulieren/open-forms-sdk/pull/71

### CODE REVIEW

File: docs/installation/config.rst [hunk 1]
```diff
@@ -96,6 +96,9 @@ Log settings
 * ``SENTRY_DSN``: URL of the sentry project to send error reports to. Defaults
   to an empty string (ie. no monitoring). See `Sentry settings`_.
 
+* ``SDK_SENTRY_DNS``: URL of the sentry project for the SDK to send error reports to. Defaults
```
Comment (sergei-maertens) [inline docs/installation/config.rst:99]:
```suggestion
* ``SDK_SENTRY_DSN``: URL of the sentry project for the SDK to send error reports to. Defaults
```

File: docs/installation/config.rst [hunk 2]
```diff
@@ -96,6 +96,9 @@ Log settings
...
+  to an empty string (ie. no monitoring). This is a **public** Sentry DNS. See `Sentry settings`_.
```
Comment (sergei-maertens) [inline docs/installation/config.rst:100]:
```suggestion
  to an empty string (ie. no monitoring). This is a **public** Sentry DSN. See `Sentry settings`_.
```

**Review by sergei-maertens — CHANGES_REQUESTED** [review]

**Comment by joeribekker** [comment]
And make sure the environment is provided as per sdk comment

File: src/openforms/conf/base.py [hunk 3]
```diff
@@ -632,6 +632,10 @@
         **SENTRY_CONFIG, integrations=get_sentry_integrations(), send_default_pii=True
     )
 
+# Sentry for the Open-Forms SDK
+SDK_SENTRY_DNS = config("SDK_SENTRY_DNS", None)
```
[the remark this answers is not in the document]
Comment (SilviaAmAm) [reply src/openforms/conf/base.py:636]: Ah, should I also modify `SENTRY_DSN = config("SENTRY_DSN", None)` to use an empty string?

[the remark this answers is not in the document]
Comment (sergei-maertens) [reply src/openforms/conf/base.py:636]: yeah, `default=None` is not the same as `default=""` :)

**Review by sergei-maertens — APPROVED** [review]

**Pull request merged**